crates/rolldown/src/utils/parse_to_ecma_ast.rs

```diff
@@ -52,8 +52,8 @@ pub fn parse_to_ecma_ast(
     ModuleType::Ts => (source.try_into_string()?, OxcParseType::Ts),
     ModuleType::Tsx => (source.try_into_string()?, OxcParseType::Tsx),
     ModuleType::Css => {
```

Comment:
The implementation is different since our CSS implementation is not the same as the esbuild https://github.com/evanw/esbuild/blob/d34e79e2a998c21bb71d57b92b0017ca11756912/internal/bundler/bundler.go#L278-L281